src/test/scala/nl/knaw/dans/easy/fedora2vault/AppSpec.scala

```diff
@@ -171,8 +153,30 @@ class AppSpec extends TestSupportFixture with MockFactory with FileSystemSupport
         |</files>""".stripMargin
   }
 
-  private def csvPrinter(sb: JavaStringBuilder): CSVPrinter = {
-    CsvRecord.csvFormat.print(sb)
+  it should "report invalid file metadata" in {
+    val app = new MockedApp()
+    implicit val fedoraProvider: FedoraProvider = app.fedoraProvider
+    expectedAudiences(Map(
+      "easy-discipline:6" -> "D35400",
+    ))
+
+    expectAUser(app.ldapContext)
+    expectedFoXmls(
+      app.fedoraProvider,
+      sampleFoXML / "streaming.xml",
+      (testDir / "easy-file-35.xml").writeText(
+        (sampleFoXML / "easy-file-35.xml").contentAsString.split("\n")
+          .filterNot(_.contains("<visibleTo>")).mkString("\n")
+      ),
+    )
+    expectedSubordinates(app.fedoraProvider, "easy-file:35")
+    expectedManagedStreams(app.fedoraProvider,
+      (testDir / "something.txt").writeText("don't care")
+    )
+
+    app.simpleTransform("easy-dataset:13", testDir / "bags" / UUID.randomUUID.toString) should matchPattern {
+      case Failure(e) if e.getMessage == "No <visibleTo> in EASY_FILE_METADATA for easy-file:35" =>
```

Comment:
Why am I having trouble with 

```suggestion
    app.simpleTransform("easy-dataset:13", testDir / "bags" / UUID.randomUUID.toString) shouldBe Failure(new Exception("No <visibleTo> in EASY_FILE_METADATA for easy-file:35"))
```
![image](https://user-images.githubusercontent.com/10553630/82037037-ea1ea780-96a1-11ea-8b58-aed3d9a3bfa8.png)